ArjunaCore/arjuna/tests/classes/com/hp/mwtests/ts/arjuna/atomicaction/TxControlUnitTest.java

```diff
@@ -56,4 +58,13 @@ public void testGetSet () throws Exception
         
         assertTrue(TxControl.getXANodeName().equals(nodeName));
     }
+
+    @Test
+    public void testNodeName () {
+        try {
+            TxControl.setXANodeName(new String(new char[NODE_NAME_SIZE + 1]));
+        } catch (IllegalArgumentException e) {
+            assertNotNull("IllegalArgumentException has no detail message.", e.getMessage());
```

Comment:
Thanks @mmusgrov for your PR! Could we use the assertEqual rather then assertNotNull now?

Reply:
I don't see why we would need to do that, so no.

Reply:
We could change it when we move to junit 5

Reply:
Ok, I will approve it then.

Reply:
I think you are looking at the [javadoc for junit.framework](https://javadoc.io/static/junit/junit/4.13.2/junit/framework/Assert.html) which has been deprecated in favour of org.junit which is the "new" framework.

Reply:
Actually I was referring to org.junit. But I wrote assertEqual whilst it is [assertEquals](https://junit.org/junit4/javadoc/4.13/org/junit/Assert.html#assertEquals(java.lang.String,%20java.lang.Object,%20java.lang.Object)) with the final 's'. It was already imported at line 8, now 9.